motionSetPointsApp/src/setPoint.cpp

```diff
@@ -140,14 +140,14 @@ void loadFile(FileIOInterface *fileIO, const char *fname, const char *env_fname,
                 LookupRow row = createRowFromFileLine(line);
                 int numberOfCoordsInLine = row.coordinates.size();
                 if (numberOfCoordsInLine == 0) {
-                    throw std::runtime_error("Error parsing " + std::string(fname) + " line " + std::to_string(table.rows.size() + 1) + ": " + line);
+                    throw std::runtime_error("Error parsing " + std::string(fname) + " line " + std::to_string((long double)table.rows.size() + 1) + ": " + line);
                 }
```

Comment:
It seems slightly strange casting an integer type to a `long double`, does  `unsigned long long`  not work?

Reply:
My bad, this is what happens when I try and put something in late at night